src/ontology/mondo-edit.obo

```diff
@@ -480062,25 +480061,22 @@ is_a: MONDO:0019701 {source="Orphanet:99645", source="icd11.foundation:206703263
 
 [Term]
 id: MONDO:0020474
-name: cheirospondyloenchondromatosis
-def: "Cheirospondyloenchondromatosis is an extremely rare type of enchondromatosis of very early onset (from neonatal period to infancy) characterized by symmetrical multiple enchondromas with metacarpal and phalangeal involvement resulting in short hands and feet, platyspondyly, mild to moderate short stature and intellectual disability." [Orphanet:99647]
+name: obsolete cheirospondyloenchondromatosis
+def: "OBSOLETE. Cheirospondyloenchondromatosis is an extremely rare type of enchondromatosis of very early onset (from neonatal period to infancy) characterized by symmetrical multiple enchondromas with metacarpal and phalangeal involvement resulting in short hands and feet, platyspondyly, mild to moderate short stature and intellectual disability." [Orphanet:99647]
 comment: Reason of obsoletion: out of scope - MONDO:excludeHistoricalDisease. Term to consider: -
-subset: gard_rare {source="GARD:19682", source="MONDO:GARD"}
-subset: nord_rare {source="MONDO:NORD"}
-subset: obsoletion_candidate
-subset: rare
 synonym: "generalised enchondromatosis with platyspondyly" EXACT OMO:0003005 []
 synonym: "generalized enchondromatosis with platyspondyly" EXACT []
 xref: GARD:19682 {source="MONDO:GARD"}
-xref: icd11.foundation:1707937102 {source="MONDO:equivalentTo"}
-xref: MEDGEN:1390387 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
-xref: Orphanet:99647 {source="MONDO:equivalentObsolete"}
-xref: SCTID:725104005 {source="MONDO:equivalentTo"}
-xref: UMLS:C4510810 {source="MEDGEN:1390387", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
-is_a: MONDO:0018230 {source="Orphanet:99647"} ! skeletal dysplasia
+xref: icd11.foundation:1707937102 {source="MONDO:obsoleteEquivalent"}
+xref: MEDGEN:1390387 {source="MONDO:obsoleteEquivalent", source="MONDO:MEDGEN"}
+xref: Orphanet:99647 {source="MONDO:obsoleteEquivalent"}
```

Comment:
It looks like this was also obsoleted in [Orphanet](https://www.orpha.net/en/disease/detail/99647). 
Therefore the source should be "MONDO:obsoleteEquivalentObsolete"